GitUI/ScriptsEngine/ScriptsManager.ScriptRunner.cs

```diff
@@ -139,7 +144,13 @@ private static CommandStatus RunScriptInternal(ScriptInfo script, IWin32Window o
                     }
                 }
 
-                return new CommandStatus(executed: true, needsGridRefresh: !script.RunInBackground);
+                bool needsGridRefresh = !script.RunInBackground;
+                if (needsGridRefresh)
```

Comment:
Tech debt in one? `needsGridRefresh` is created, and immediately made unnecessary. Unless this is intentional.

Reply:
It was kind of intentional to keep the formerly needed variable. Removed.